### PR TITLE
API-5181: add choice page to users with no applications

### PR DESCRIPTION
**Pull request by mattclark-zerogravit**

### CODE REVIEW

File: app/uk/gov/hmrc/thirdpartydeveloperfrontend/controllers/forms.scala [hunk 1]
```diff
@@ -533,3 +533,13 @@ object SelectTopicsFromSubscriptionsForm {
     )(SelectTopicsFromSubscriptionsForm.apply)(SelectTopicsFromSubscriptionsForm.unapply)
   )
 }
+
+final case class NoApplicationsChoiceForm(choice: Option[String])
```
Comment (AndySpaven) [inline app/uk/gov/hmrc/thirdpartydeveloperfrontend/controllers/forms.scala:537]: We're putting these in companion objects of controllers nowadays

**Review by syed-ali-hmrc — APPROVED** [review]

File: app/uk/gov/hmrc/thirdpartydeveloperfrontend/controllers/ManageApplications.scala [hunk 2]
```diff
@@ -17,46 +17,42 @@
...
-
-    appsByTeamMember: AppsByTeamMemberService,
-    upliftLogic: UpliftLogic,
-    
-    manageApplicationsView: ManageApplicationsView,
-    addApplicationSubordinateEmptyNestView: AddApplicationSubordinateEmptyNestView,
-    mcc: MessagesControllerComponents
-)(implicit val ec: ExecutionContext, val appConfig: ApplicationConfig, val environmentNameService: EnvironmentNameService)
-   extends LoggedInController(mcc) {
+class ManageApplications @Inject()(
```
Comment (AndySpaven) [inline app/uk/gov/hmrc/thirdpartydeveloperfrontend/controllers/ManageApplications.scala:33]: Can you sort the formatting please.  Really object to things been this indented

Comment (mattclark-zerogravit) [reply app/uk/gov/hmrc/thirdpartydeveloperfrontend/controllers/ManageApplications.scala:33]: will do

File: app/uk/gov/hmrc/thirdpartydeveloperfrontend/controllers/forms.scala (hunk 1 above)
Comment (mattclark-zerogravit) [reply app/uk/gov/hmrc/thirdpartydeveloperfrontend/controllers/forms.scala:537]: will do

File: app/uk/gov/hmrc/thirdpartydeveloperfrontend/controllers/ManageApplications.scala [hunk 3]
```diff
@@ -17,46 +17,42 @@
...
+                                    upliftLogic: UpliftLogic,
+                                    manageApplicationsView: ManageApplicationsView,
+                                    mcc: MessagesControllerComponents
+                                  )(implicit val ec: ExecutionContext, val appConfig: ApplicationConfig, val environmentNameService: EnvironmentNameService)
+  extends LoggedInController(mcc) {
 
   def manageApps: Action[AnyContent] = loggedInAction { implicit request =>
     for {
-      upliftData                  <- upliftLogic.aUsersSandboxAdminSummariesAndUpliftIds(request.userId)
+      upliftData <- upliftLogic.aUsersSandboxAdminSummariesAndUpliftIds(request.userId)
```
[the remark this answers is not in the document]
Comment (peteslater-ee) [reply app/uk/gov/hmrc/thirdpartydeveloperfrontend/controllers/ManageApplications.scala]: There is a way to have it line up the arrows and equals in a for comp

https://scalameta.org/scalafmt/docs/configuration.html#alignpresetmost

But I'm starting to think it's maybe not worth it :(

**Review by peteslater-ee — APPROVED** [review]

**Pull request merged**